### PR TITLE
Update renku UI 3.4.3

### PR DESCRIPTION
**Pull request by lorenzo-cavazzi**
Update renku UI to version 3.4.3
/deploy #persist

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,5 +1,41 @@
...
+**🐞 Bug Fixes**
+
+- **UI**: Prevent flashing a spinning wheel when loading workflows
+  (`#2493 <https://github.com/SwissDataScienceCenter/renku-ui/pull/2493>`_).
+- **UI**: Restore the flag to show inactive workflows
+  (`#2502 <https://github.com/SwissDataScienceCenter/renku-ui/pull/2502>`_).
+- **UI**: Prevent infinite spinning wheel when accessing projects by numeric ID
+  without the required user's permissions
+  (`#2476 <https://github.com/SwissDataScienceCenter/renku-ui/pull/2476>`_).
+- **UI**: Open the target notebook when accessing a session from the file preview
```
Comment (rokroskar) [inline CHANGELOG.rst:17]: what was fixed here? If I'm interpreting this correctly, can we say:

```suggestion
- **UI**: Fix an issue where a notebook would not automatically open in a session for anonymous users
```

Comment (lorenzo-cavazzi) [reply CHANGELOG.rst:17]: Ops I used a very cryptic message. Thanks for spotting it. The suggestion is on point :ok_hand:

**Comment by rokroskar** [comment]
Because you are missing a top-level heading, e.g.

```rst
0.25.4
------
```

File: CHANGELOG.rst [hunk 2]
```diff
@@ -3,21 +3,45 @@
 0.25.4
 ------
 
-Renku ``0.25.4`` brings bug fixes for KG.
+Renku ``0.25.4`` introduces several KG and UI big-fixes.
```
Comment (rokroskar) [inline CHANGELOG.rst:6]:
```suggestion
Renku ``0.25.4`` introduces several KG and UI bug-fixes.
```

**Review by rokroskar — CHANGES_REQUESTED** [review]

**Review by rokroskar — APPROVED** [review]

**Review by aledegano — APPROVED** [review]

File: CHANGELOG.rst (hunk 2 above)
Comment (lorenzo-cavazzi) [reply CHANGELOG.rst]: :pray:

**Pull request merged**